### ISSUE TITLE
storybook is unable to load antd components styling

### ISSUE THREAD

**Issue by kaiz123** [GroceriStar/antd-showcase-components#25]
Webpack.config.js inside .storybook folder is configured but style from antd components still does not load.

**Comment by atherdon** [comment]
Yes, I think because we need to add some lines inside of webpack. I'm not very familiar with webpack, but maybe you can google if people have the same issue as we have

**Comment by kaiz123** [comment]
I googled and came to the best possible solution. Still seems like some issue exists

**Comment by atherdon** [comment]
you can add this an issue at antd repository. maybe it will help other people too. Guys from antd team solving things very quickly - i know

**Comment by kaiz123** [comment]
Okay that's a good suggestion. Will do that.

**Comment by atherdon** [comment]
i think this is how we can give something back. because they did a great job

**Comment by atherdon** [comment]
you can add here issue that create at webpack

**Comment by atherdon** [comment]
I think one of the reasons of this issue can be version of babel. I find out, that you add `  "babel-preset-es2015"` and this is a module, that requre an old `babel-core v6` plugin. Right now Babel team moved to a whole new approach and replace `babel-core` to `@babel\core` etc... maybe this is causing an issue...
